api/app/schemas/sfms.py

```diff
@@ -0,0 +1,12 @@
+""" This module contains pydantic schemas related to SFMS.
+"""
+from typing import List
+from pydantic import BaseModel
+
+class HourlyTIF(BaseModel):
+    """ URL of the public hourly tif and it's last modified date in UTC """
```

Comment:
Did you remove the last modified date?

Reply:
I did, thanks for catching, updated in https://github.com/bcgov/wps/pull/3575/commits/e40e26b0a4cd31e4a5a19cf70c9e29fac00ca234